README.rst

```diff
@@ -792,15 +761,12 @@ statement sequence can then be aborted and as such the ``something_else`` call
 needs no code generation or consideration anymore.
 
 To that end, Nuitka works with a special node that raises an exception and is
-wrapped with a so called "side_effects" expression, but yet can be used in code
+wrapped with a so-called "side_effects" expression, but yet can be used in the code
 as an expression having a value.
 
 .. admonition:: Status
 
-   The propagation of exceptions is mostly implemented, but needs handling in
-   every kind of operations, and not all of them might do it already. As work
-   progresses or examples arise, the coverage will be extended. Feel free to
-   generate bug reports with non-working examples.
+   The propagation of exceptions is mostly implemented but needs handling in every kind of operations, and not all of them might do it already. As work progresses or examples arise, the coverage will be extended. Feel free to generate bug reports with non-working examples.
```

Comment:
The indentation is inconsistent here, maybe this is an error in ReST? Can you check with `restructuretext-lint` ?

Reply:
Sure @kayhayen